packages/webpack-cli/__tests__/arg-parser.test.js

```diff
@@ -448,4 +448,17 @@ describe('arg-parser', () => {
         });
         expect(warnMock.mock.calls.length).toEqual(0);
     });
+
+    it('parses nested multiType flag', () => {
+        const res = argParser(basicOptions, ['--env', 'a.b=d', '--env', 'a.b.c=d'], true);
```

Comment:
Lets add one more case for 
```suggestion
        const res = argParser(basicOptions, ['--env', 'a.b=c', '--env', 'd.e.f=g'], true);
```

Reply:
yes

Reply:
we should also add multi compiler I think

Reply:
wip

Reply:
multicompiler not in scope, added more cases